test/langtools/ProblemList.txt

```diff
@@ -57,18 +57,6 @@ tools/javac/modules/SourceInSymlinkTest.java
 
 tools/javap/output/RepeatingTypeAnnotations.java                                8057687    generic-all    emit correct byte code an attributes for type annotations
 
-###########################################################################
-#
-# sjavac
-
-tools/sjavac/IncCompileFullyQualifiedRef.java                                   8152055    generic-all    Requires dependency code to deal with in-method dependencies.
-tools/sjavac/IncCompileWithChanges.java                                         8152055    generic-all    Requires dependency code to deal with in-method dependencies.
-
-tools/sjavac/ApiExtraction.java                                                 8158002    generic-all    Requires investigation
-tools/sjavac/IgnoreSymbolFile.java                                              8158002    generic-all    Requires investigation
-tools/sjavac/ClasspathDependencies.java                                         8158002    generic-all    Requires investigation
```

Comment:
I presume these issues, 8152055, 8158002, are still open?    They should either be closed or assigned the the build team.

Reply:
No, I've closed them already, with a reference to this issue.

Reply:
To be clear: they referred to the test code that is removed as part of this PR, so closing them is the only reasonable thing to do.